CHANGES.rst

```diff
@@ -2,6 +2,14 @@
 Changes for Crate Admin Interface
 =================================
 
+Unreleased
+==========
+
+Fixes
+-----
+
+- Updated the Tweet tutorial table to no longer use deprecated column types.
```

Comment:
I am wondering whether it is a fix.

Reply:
Do you think it counts more as a 'change'?